### PR TITLE
encryption service done

### PR DESCRIPTION
**Pull request by sandeep-groww**
## What does this PR do?
An encryption service that will be used by the consumers  to encrypt and decrypt the data 

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [X] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist before merging

_Put an `x` in the boxes that apply_

- [ ] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

File: packages/encryption/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "devDependencies": {
+    "@groww-tech/eslint-config": "^1.0.4",
+    "@groww-tech/tsconfig": "^1.0.0",
+    "@types/crypto-js": "^4.1.1",
+    "crypto-js": "^3.3.0",
+    "eslint": "7.26.0",
+    "typescript": "4.4.4"
+  },
+  "dependencies": {
+    "crypto-js": "^3.3.0"
```
Comment (harshitpruthi94) [inline packages/encryption/package.json:49]: remove caret

File: packages/encryption/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  };
+
+  try {
+    const ciphertext = aes.encrypt(JSON.stringify(object), secretKey);
+
+    response.data = ciphertext.toString();
+
+    return response;
+
+  } catch (e: any) {
```
Comment (harshitpruthi94) [inline packages/encryption/src/index.ts:42]: remove any

File: packages/encryption/.npmignore [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# OSX
+.DS_Store
+
```
Comment (shubhamgavali1998) [inline packages/encryption/.npmignore:3]: add node_modules

File: packages/encryption/.gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# build
+dist
```
Comment (shubhamgavali1998) [inline packages/encryption/.gitignore:2]: add node_modules

File: packages/encryption/package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/Groww/webster/issues"
+  },
+  "homepage": "https://github.com/Groww/webster/tree/develop/packages/encryption#readme",
+  "devDependencies": {
+    "@groww-tech/eslint-config": "^1.0.4",
```
Comment (shubhamgavali1998) [inline packages/encryption/package.json:45]: use `workspace:*`

File: packages/encryption/src/index.ts [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+ * ```
+ * encrypt('Data to encrypt','secret-key')); => Output will be
+ * {
+ *  data:'##Some random ciphered text',
+ *  error:null
+ * }.
+ * ```
+ */
+
+export const encrypt = (object:object, secretKey:string) => {
```
Comment (shubhamgavali1998) [inline packages/encryption/src/index.ts:27]: rename object

File: packages/encryption/src/index.ts [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+  const response:ResponseType = {
```
Comment (shubhamgavali1998) [inline packages/encryption/src/index.ts:29]: remove ResponseType

**Pull request merged**